<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>321</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="PerformanceTab_2">
     <attribute name="title">
      <string>Intro</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>531</width>
        <height>251</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Process: This is a process scheduler which allows you to view system process Information and manage processes.

Multithread Chatroom: This is a TCPServer which allows multiple clients to connect, the content sent by the client can be shown in the display window</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="processTab">
     <attribute name="title">
      <string>Process</string>
     </attribute>
     <widget class="QListWidget" name="processList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Processes:</string>
      </property>
     </widget>
     <widget class="QLabel" name="nPSleepLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="nPActiveLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="killButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill Process</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sleeping Processes:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Running Processes:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Zombie Processes:</string>
      </property>
     </widget>
     <widget class="QLabel" name="nPLabel">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>478</x>
        <y>220</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Process</string>
      </property>
     </widget>
     <widget class="QLabel" name="nPZombieLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>MultiThreadChatRoom</string>
     </attribute>
     <widget class="QListWidget" name="chatRoom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>471</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>processList</tabstop>
  <tabstop>tabs</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>killButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
